File: app/src/main/java/com/tt/handsomeman/response/StandardResponse.java
package com.tt.handsomeman.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StandardResponse {
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("statusCode")
    @Expose
    private String statusCode;
    @SerializedName("message")
    @Expose
    private String message;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(String statusCode) {
        this.statusCode = statusCode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/tt/handsomeman/response/StandardResponse.kt
package com.tt.handsomeman.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

open class StandardResponse {
    @SerializedName("status")
    @Expose
    var status: String? = null
    @SerializedName("statusCode")
    @Expose
    var statusCode: String? = null
    @SerializedName("message")
    @Expose
    var message: String? = null
}
